Backpack/src/main/java/net/skyscanner/backpack/text/BpkText.kt

```diff
@@ -66,7 +71,7 @@ open class BpkText @JvmOverloads constructor(
     HEAVY
   }
 
-  @IntDef(XS, SM, BASE, LG, XL, XXL, XXXL, CAPS)
+  @IntDef(XS, SM, BASE, LG, XL, XXL, XXXL, CAPS, HERO1, HERO2, HERO3, HERO4, HERO5)
```

Comment:
How do the old enum values combine with heading3/footnote etc?

Reply:
That will happen in a later change - the plan is to deprecate all the existing styles with the new ones being added. It's the same process we follow on iOS & Web as well